src/browser/app/profile/zen-browser.js

```diff
@@ -53,15 +58,63 @@ pref('browser.toolbars.bookmarks.visibility', 'never');
 // Enable Do Not Track and GPC by default.
 pref("privacy.donottrackheader.enabled", true);
 pref("privacy.globalprivacycontrol.enabled", true);
+
 // Disable more telemetry
 pref("toolkit.telemetry.enabled", false);
 pref("browser.ping-centre.telemetry", false);
 pref("browser.attribution.enabled", false);
 pref("toolkit.telemetry.pioneer-new-studies-available", false);
+pref("toolkit.telemetry.cachedClientID", "");
+pref("toolkit.telemetry.previousBuildID", "");
+pref("toolkit.telemetry.server_owner", "");
+pref("toolkit.coverage.opt-out", true);
+pref("toolkit.coverage.enabled", false);
+pref("toolkit.coverage.endpoint.base", "");
+pref("toolkit.crashreporter.infoURL", "");
+pref("security.protectionspopup.recordEventTelemetry", false);
+
+// Query stripping - ported from Brave (https://github.com/brave/brave-core/blob/master/components/query_filter/utils.cc#L23)
+pref("privacy.query_stripping.strip_list", "__hsfp __hssc __hstc __s _branch_match_id _branch_referrer _gl _hsenc _kx _openstat at_recipient_id at_recipient_list bbeml bsft_clkid bsft_uid dclid et_rid fb_action_ids fb_comment_id fbclid gbraid gclid guce_referrer guce_referrer_sig hsCtaTracking irclickid mc_eid ml_subscriber ml_subscriber_hash msclkid mtm_cid oft_c oft_ck oft_d oft_id oft_ids oft_k oft_lk oft_sk oly_anon_id oly_enc_id pk_cid rb_clickid s_cid ss_email_id twclid unicorn_click_id vero_conv vero_id vgo_ee wbraid wickedid yclid ymclid ysclid")
+
+// Disables fetching and updating safebrowsing lists from Google
+pref("browser.safebrowsing.malware.enabled", false);
+pref("browser.safebrowsing.phishing.enabled", false);
+pref("browser.safebrowsing.blockedURIs.enabled", false);
```

Comment:
And how is one supposed to enable this given they are not in the settings?

Reply:
What you're saying is that theres no need to disable them because they cant be enabled? I dont get it.

Reply:
...it's just a question